packages/libs/react-ui/src/components/Accordion/AccordionHeading.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+  accordionButtonClass,
+  accordionHeadingTitleClass,
+  accordionToggleIconClass,
+} from './Accordion.css';
+
+import { SystemIcon } from '@components/Icon';
+import classNames from 'classnames';
+import type { FC } from 'react';
+import React from 'react';
+
+export interface IAccordionHeadingProps {
+  icon: keyof typeof SystemIcon;
```

Comment:
In what instance would the icon change? The rotation only works for the arrow or the + so I don't think we want to give them the option to use any icon. I would just stick with a fixed + for now

Reply:
It's because I originally wanted to re-use the heading for the NavAccordion group headings but too many little things differ. I think I'll just get rid of the whole Heading component

Reply:
Done